pkg/idtools/idtools.go

```diff
@@ -303,3 +303,11 @@ func parseSubidFile(path, username string) (ranges, error) {
 	}
 	return rangeList, nil
 }
+
+func NewIDPairIdentity(IdPair IDPair) Identity {
+	return Identity{IdType: TypeIDPair, IdPair: IdPair}
+}
```

Comment:
This utility function seems fine, but I noticed most callers also initialize `IdPair` inline. Did you consider accepting `UID` and `GID` instead of `IdPair`, or adding yet another utility function?